grails5 upgrade - geb issue

niravassar opened the pull request.

Commit:
grails5 upgrade

sdelamo commented on complete/build.gradle:
```diff
@@ -5,8 +5,8 @@ buildscript {
     }
     dependencies {
         classpath "org.grails:grails-gradle-plugin:$grailsVersion"
```
this should be decoupled from grails Version

sdelamo commented on complete/build.gradle:
```diff
@@ -58,35 +58,38 @@ dependencies {
 //tag::webdriverBinaries[]
 webdriverBinaries {
-    chromedriver '78.0.3904.105'  // <6>
+    chromedriver {
```
this should be the same version as grails_i18n guide.

sdelamo commented on complete/build.gradle:
```diff
@@ -58,35 +58,38 @@ dependencies {
+        version = '78.0.3904.105'
+        architecture = "X86"
+    }  // <6>
     geckodriver '0.26.0' // <7>
```
this should be the same version as grails_i18n guide

sdelamo commented on complete/gradle.properties:
```diff
@@ -1,2 +1,6 @@
-grailsVersion=4.0.1
-gormVersion=7.0.2.RELEASE
+grailsVersion=5.0.0
```
this should be 5.0.1

sdelamo commented on complete/gradle.properties:
```diff
@@ -1,2 +1,6 @@
-grailsVersion=4.0.1
-gormVersion=7.0.2.RELEASE
+grailsVersion=5.0.0
+gormVersion=7.1.0.M5
```
this should be 7.1.0

sdelamo commented on initial/build.gradle:
```diff
@@ -5,10 +5,11 @@ buildscript {
     }
     dependencies {
         classpath "org.grails:grails-gradle-plugin:$grailsVersion"
```
this should be decoupled from grails Version

sdelamo commented on initial/build.gradle:
```diff
@@ -56,34 +58,41 @@ dependencies {
 webdriverBinaries {
-    chromedriver '78.0.3904.105'
-    geckodriver '0.26.0'
+    chromedriver {
```
this should be the same version as grails_i18n

sdelamo commented on initial/build.gradle:
```diff
@@ -56,34 +58,41 @@ dependencies {
+        version = '78.0.3904.105'
+        architecture = "X86"
+    }  // <6>
+    geckodriver '0.26.0' // <7>
```
this should be the same version as grails_i18n

sdelamo commented on initial/gradle.properties:
```diff
@@ -1,2 +1,6 @@
-grailsVersion=4.0.0.BUILD-SNAPSHOT
-gormVersion=7.0.0.M1
+grailsVersion=5.0.0
```
5.0.1

sdelamo commented on initial/gradle.properties:
```diff
@@ -1,2 +1,6 @@
-grailsVersion=4.0.0.BUILD-SNAPSHOT
-gormVersion=7.0.0.M1
+grailsVersion=5.0.0
+gormVersion=7.1.0.M5
```
7.1.0

jamesfredley committed:
Update README to point at canonical guide on grails.apache.org (refs apache/grails-static-website#354)

jamesfredley committed:
Replace dead publisher CI with sample-app gradle check workflow

Delete: githubactions-build.sh
Add: .github/workflows/gradle.yml (auto-detects Java from Gradle wrapper version)

(refs apache/grails-static-website#354)
